src/cycles/index.ts

```diff
@@ -0,0 +1,4 @@
+import { addCyclesBack } from './add-cycles-back';
+import { removeCycles } from './remove-cycles';
```

Comment:
Potential alternative names for `remove` and `addBack` (just ideas, no strong preference):

`cutCycles`
`reconnectCycles`

Reply:
You're right and I like those names